portfolio/src/main/webapp/script.js

```diff
@@ -27,9 +27,21 @@ function addRandomFact() {
   factContainer.innerText = fact;
 }
 
-// Add greeting to the page
-function getGreeting() {
-  fetch('/data').then(response => response.text()).then((greeting) => {
-    document.getElementById('greeting-container').innerText = greeting;
-  });
+/**
+ * Fetch the list of comments and add them to the page
+ */
+function getComments(){
+fetch('/comments').then(response => response.json()).then((comments) => {
+const commentContainer = document.getElementById('comments-container');
+comments.forEach(function(comment) {
+commentContainer.appendChild(createListElement(comment))
+});
+});
 }
+
+/** Creates an <li> element containing text. */
+function createListElement(text) {
+  const liElement = document.createElement('li');
```

Comment:
Let's be consistent with quotes. Either use single quotes everywhere or double quotes everywhere.

Reply:
shouldn't it be /**/ for functions and // for regular comments?

Reply:
im referring to quotes, not the slashes. See line 44. You have "createElement('li');". 
In other places, you use double ("") quotes instead of (''). Pick one. For each language, use consistent semantics. 
The javascript style guide recommends single quotes: 
https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes

Looking through this code again, it seems you already use single quotes everywhere. So disregard.